mesh/meshdb_test.go

```diff
@@ -975,3 +975,24 @@ func TestMesh_FindOnce(t *testing.T) {
 		}
 	})
 }
+
+func BenchmarkGetBlockHeader(b *testing.B) {
+	// blocks is set to be twice as large as cache to avoid hitting the cache
+	cache := layerSize
+	blocks := make([]*types.Block, cache*2)
+	// 1 is passed as a cache size because it is actually multiplied by layerSize
```

Comment:
I don't understand this comment without context. Why is it multiplied by layer size?

Reply:
internally size of the cache will is multiplied by a constant (layerSize = 200). will update the comment